package.json

```diff
@@ -1,10 +1,13 @@
 {
   "name": "eslint-config-jquery",
-  "version": "3.0.2",
+  "version": "4.0.0",
```

Comment:
Please don't change this. It'll get updated on the next release.

Reply:
Sorry about that :)